cmd/lakectl/cmd/doctor.go

```diff
@@ -11,11 +11,42 @@ import (
 	"github.com/treeverse/lakefs/pkg/api"
 )
 
-var (
-	ErrCredential       = errors.New("credential error")
-	ErrWrongEndpointURI = errors.New("wrong endpoint_uri error")
-	ErrUnknownConfig    = errors.New("unknown configuratioin error")
-)
+type ErrCredential struct {
+	Message string
+	Details string
+}
+
+func (e *ErrCredential) Error() string {
+	return (e.Message + "\n" + e.Details + "\n")
+}
+
+type ErrWrongEndpointURI struct {
+	Message string
+	Details string
+}
+
+func (e *ErrWrongEndpointURI) Error() string {
+	return (e.Message + "\n" + e.Details + "\n")
+}
+
+type ErrUnknownConfig struct {
+	Message string
+	Details string
+}
+
+func (e *ErrUnknownConfig) Error() string {
+	return (e.Message + "\n" + e.Details + "\n")
+}
+
+type SuccessMessage struct {
+	Message string
+}
+
+var configErrorTemplate = `{{ .Message |red }}
+{{  .Details | red }}
```

Comment:
```suggestion
{{  .Details }}
```

I think the details are less important, and templates let us say that!